sycl/test-e2e/Matrix/joint_matrix_int8_rowmajorA_rowmajorB_impl.hpp

```diff
@@ -51,7 +51,6 @@ void matrix_multiply(big_matrix<TResult, M, N> &C, big_matrix<T, M, K> &A,
                  accA.template get_multi_ptr<access::decorated::no>() +
                      (sg_startx * TM) * K + k * TK,
                  K);
-             // VNNI transform is done automatically at this level
```

Comment:
please, keep this comment, it explains that VNNI is still happening, it is just hidden